xapian-applications/omega/templates/query

```diff
@@ -37,87 +37,96 @@ $if{$opt{topterms},$include{inc/toptermsjs}}
+<fieldset class="query-search">
+<input class="query-search" id="omega-autofocus" type=search name=P value="$html{$query}" placeholder="Search..." size=65 autofocus>
 <script>
 if (!("autofocus" in document.createElement("input")))
  document.getElementById("omega-autofocus").focus();
 </script>
-<input type=submit value="Search">
-<hr>
-$if{,$include{inc/anyalldropbox},$include{inc/anyallradio}}
+<input class="query-search" type=submit value="Search">
+</fieldset>
+${Todo: Replace radios with search buttons}
+$if{,$include{inc/anyalldropbox},
+<fieldset class="query-options">$include{inc/anyallradio}</fieldset>}
+${Suggestions - shows suggested terms to improve query}
 $if{$opt{topterms},
- <div title="Suggested terms to add to your query"
+ <fieldset class="query-suggested-container" title="Suggested terms to add to your query"
  style="text-align:left;background:#cfc;border:1px solid green;padding:2px;font:11px verdana$. arial$. helvetica$. sans-serif;">
- $map{$topterms,<span style="white-space:nowrap"><input type=checkbox name=X value="$html{$prettyterm{$_}}" onClick="C(this)">$html{$prettyterm{$_}}</span> }
- <br><noscript><input type=hidden name=ADD value=1></noscript>
- </div>
+ $map{$topterms,<span class="query-suggested-element" style="white-space:nowrap"><input type=checkbox name=X value="$html{$prettyterm{$_}}" onClick="C(this)">$html{$prettyterm{$_}}</span> }
+ <noscript><input type=hidden name=ADD value=1></noscript>
+ </fieldset>
 }
-<br>
-$if{$suggestion,Did you mean: <a href="$html{$or{$env{SCRIPT_NAME},omega}?DB=$url{$dbname}&P=$url{$suggestion}$if{$ne{$hitsperpage,10},&HITSPERPAGE=$hitsperpage}$if{$fmt,&FMT=$url{$fmt}}$if{$cgi{COLLAPSE},&COLLAPSE=$url{$cgi{COLLAPSE}}}$map{$relevants,&R=$_}$if{$cgi{THRESHOLD},&THRESHOLD=$url{$cgi{THRESHOLD}}}$map{$cgilist{B},&B=$url{$_}}&SPELL=0}">$html{$suggestion}</a><br>}
+$if{$suggestion,Did you mean: <a class="query-suggestion" href="$html{$or{$env{SCRIPT_NAME},omega}?DB=$url{$dbname}&P=$url{$suggestion}$if{$ne{$hitsperpage,10},&HITSPERPAGE=$hitsperpage}$if{$fmt,&FMT=$url{$fmt}}$if{$cgi{COLLAPSE},&COLLAPSE=$url{$cgi{COLLAPSE}}}$map{$relevants,&R=$_}$if{$cgi{THRESHOLD},&THRESHOLD=$url{$cgi{THRESHOLD}}}$map{$cgilist{B},&B=$url{$_}}&SPELL=0}">$html{$suggestion}</a>}
+${Query metadata - shows query performance or document size}
 $or{$html{$error},
+<p class="query-metadata">
 $if{$eq{$msize,0},
 $if{$query,No documents match your query,
-<hr>Searching $nice{$dbsize} documents
-},
+Searching $nice{$dbsize} documents
+}</p>,
 $if{$not{$msizeexact},
  $nice{$add{$topdoc,1}}-$nice{$last} of about $nice{$msize} matches,
  $if{$and{$eq{$last,$msize},$eq{$topdoc,0}},
   All $nice{$msize} matches,
   $nice{$add{$topdoc,1}}$if{$ne{$add{$topdoc,1},$last},-$nice{$last}} of exactly $nice{$msize} matches}
 }
-<hr>
-</center>
-$list{$map{$queryterms,$list{$html{$uniq{$unstem{$_}}},<b>,</b>/<b>,</b>}:&nbsp;$nice{$freq{$_}}},Term frequencies: ,$. ,}
```

Comment:
Even if you want to drop the `b` elements, there's a `/` in the middle which was an output character separating the terms. The documentation for that variant of `$list{}` says:

> "$list{LIST,List ,$. ,.}" = "List 1, 2, 3, 4."

What's happening here is that `$unstem{}` returns a list (because multiple words may stem to the same term), which we want to display as either **word** or **word/alternate** (or **word/alternate/another**, and so on). This then gets the frequency of that term bolted on, and (using the outer `$list{}`) ends up as something like:

> Term frequencies: **word**: 200, **word/alternate**: 1,024

(Dropping the bolding makes sense to me, but the `/` is still important.)